config.rb.sample

```diff
@@ -29,6 +29,9 @@
 # Size of the CoreOS cluster created by Vagrant
 #$num_instances=1
 
+# Change basename of the VM
```

Comment:
Give an example of how the prefix will be applied (is a hyphen added, is there a number appended, etc)